src/pages/dashboardContainer/index.jsx

```diff
@@ -17,24 +18,30 @@ export function DashboardContainer({
   defaultStationCode,
 }) {
   // Memoize the config to prevent unnecessary re-renders
+  const [searchParams] = useSearchParams();
   const { config } = useConfig();
   const [selectedStationId, setSelectedStationId] = useState('');
   const [stations, setStations] = useState({});
   const [loading, setLoading] = useState(true);
   const [loadingChartData, setLoadingChartData] = useState(false);
 
+  const isEmbedded = searchParams.get('viewMode') === 'simple';
```

Comment:
since we renamed to simple mode, maybe update the variables to match? something like `isSimpleMode` ?